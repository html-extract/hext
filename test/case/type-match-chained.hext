<body>
  { <*:type-matches(/^(p)$|^(h[1-6])$/):type-matches(/^p|(h[1-3])$/) @text:content /> }
</body>
