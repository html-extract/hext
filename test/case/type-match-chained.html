<html>
  <head>
  </head>
  <body>
    <p>match1</p>
    <div>
      <p>match2</p>
      <P>match3</P>
      <h1>match4</h1>
      <h2>match5</h2>
      <h3>match6</h3>
      <h4>nope1</h4>
      <h5>nope2</h5>
      <h6>nope3</h6>
      <h7>nope4</h7>
    </div>
  </body>
</html>
