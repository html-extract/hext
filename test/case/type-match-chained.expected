{"content":["match1","match2","match3","match4","match5","match6"]}
